frontend/file/build.gradle

```diff
@@ -9,6 +9,7 @@ dependencies {
 
 addIntellijToCompileClasspath(withGit4Idea: true)
 applyI18nFormatterChecker()
+applySubtypingChecker()
```

Comment:
Enabled in this module just for example... I'm getting errors in `MacheteAnnotator`:

```
              getString("string.GitMachete.MacheteAnnotator.cannot-find-local-branch-in-repo").format(processedBranchName))
                                                                                                     ^
  found   : @SubtypingTop String
  required: @NonHtml String
```

which is expected currently

Reply:
okey so at the moment  all checks should throw exceptions because of format method

Reply:
Yeah... I've realized that only now, that's gonna be a tough problem to overcome. Unless an easy workaround can be found (unlikely) and Tainting Checker won't help (which I'm gonna check in a moment)... I'd just save your yesterday's findings somewhere in docs, and close the story as (mostly) `wontfix`